osu.Game/OsuGame.cs

```diff
@@ -391,11 +386,11 @@ private void load()
             // Transfer any runtime changes back to configuration file.
             SkinManager.CurrentSkinInfo.ValueChanged += skin => configSkin.Value = skin.NewValue.ID.ToString();
 
-            LocalUserPlaying.BindValueChanged(p =>
+            playingState.BindValueChanged(p =>
             {
-                BeatmapManager.PauseImports = p.NewValue;
-                SkinManager.PauseImports = p.NewValue;
-                ScoreManager.PauseImports = p.NewValue;
+                BeatmapManager.PauseImports = p.NewValue != LocalUserPlayingStates.NotPlaying;
+                SkinManager.PauseImports = p.NewValue != LocalUserPlayingStates.NotPlaying;
+                ScoreManager.PauseImports = p.NewValue != LocalUserPlayingStates.NotPlaying;
```

Comment:
Note that this stops imports from running during break time and on the fail screen, which is an extra win.